### PR TITLE
Add support for Angular 9+

### PR DESCRIPTION
**Pull request by greggbjensen**
Upgraded project to support Angular 9+

### CODE REVIEW

**Comment by beeman** [comment]
Hey @greggbjensen, thanks a lot for this contribution! I'm going to test it and publish a `next` release if it works as expected.

If this version is dropping support for Angular 8 that will mean it will be available major release so v5.

Out of curiosity, was there something not working with v4 that made you decide to upgrade it?

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,10624 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (beeman) [inline yarn.lock:1]: Could you get rid of `yarn.lock` and make sure `package-lock.json` is up-to-date with the latest in package.json?

Comment (greggbjensen) [reply yarn.lock]: The `package-lock.json` is up to date, and I removed the `yarn.lock`.

**Comment by greggbjensen** [comment]
I had to upgrade it because of the following error when trying to run with Angular 10:

```
TypeError: Cannot read property 'id' of undefined
    at registerNgModuleType 
```

```
function registerNgModuleType(ngModuleType) {
    if (ngModuleType.ɵmod.id !== null) {
        const id = ngModuleType.ɵmod.id;
        const existing = modules.get(id);
        assertSameOrNotExisting(id, existing, ngModuleType);
        modules.set(id, ngModuleType);
    }
```

**Comment by beeman** [comment]
Hi @greggbjensen - I'm surprised you have issues with Angular 10. There is a sandbox [here](https://github.com/angular-split/split-sandbox-v10) that doesn't have the issues, and the examples on Stackblitz are also using Angular 10.

**Comment by greggbjensen** [comment]
Strange, I wonder if it could be how we are using it.  I definitely works with the branch I created.

File: package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "@angular/router": "10.0.8",
     "angular-cli-ghpages": "^0.6.2",
     "bootstrap": "4.3.1",
-    "core-js": "3.6.5",
+    "core-js": "2.6.5",
```
Comment (beeman) [inline package.json:40]: Hey Gregg, what is the reason you downgraded to 2.6.5?

**Comment by beeman** [comment]
Hey @greggbjensen - yesterday I did a local test of your PR and cut a release [5.0.0-beta.0](https://www.npmjs.com/package/angular-split/v/5.0.0-beta.0) that is using your build.

Overall, it all looks very good. I'm wondering why you had to downgrade the core-js library?

File: package.json (hunk 2 above)
Comment (greggbjensen) [reply package.json:40]: That one was a mistake, I will fix it.

Comment (greggbjensen) [reply package.json:40]: Actually, I decided to keep that one the same as it was previously because the new version had errors when upgraded to it.

**Comment by beeman** [comment]
Hey @greggbjensen, finally got to merging in your changes. Thanks again for updating this library and thanks for your patience!

There will be a new release out soon with this version.

**Pull request closed without merging**